website/client/src/components/userMenu/profile.vue

```diff
@@ -833,6 +833,10 @@ export default {
   mounted () {
     this.loadUser();
     this.selectPage(this.startingPage);
+    this.$store.dispatch('common:setTitle', {
```

Comment:
this doesn't seem to update when the page is changed, I think we can simply set it into `selectPage` and it'll work if you change tab